furusiya/model/keys/callbacks.py

```diff
@@ -2,31 +2,45 @@
 from model.config import config
 
 
-def next_floor_callback():
-    Game.current_floor += 1
-
+def load_proper_area_map():
```

Comment:
Please avoid using names like `proper`, `new`, etc. that don't communicate what these functions *actually* do.